ptclient/cts/spi/rem/cl/mc.vue

```diff
@@ -100,12 +100,29 @@ export default {
       const count = this.daUniqueIDOfEachRowFromORM.length / 3
       const intValue = Math.ceil(count)
       console.log('number of slides in carousel are', count, intValue)
-      if (intValue > 1) {
-        this.dsSliderArrowVisiblity = 'always'
-      }
+
       return intValue
     },
   },
+  watch: {
+    /*
+      Problem: Sometimes an error comes 'Handling unexpected side effect in computed properties'
+      Solution: you should not edit other data in computed property, you should use watch instead
+      Ref: https://stackoverflow.com/questions/53757107/handling-unexpected-side-effect-in-computed-properties-vuejs
+    */
+    getNumOfCarouselSlides: {
```

Comment:
This shiould have a better name. Because this is not returnuing not the number of slides.